dls-controls/malcolmjs
# Feature/multi select #93

## Description

Fixed the link rendering so it redraws when both blocks move in unison.
Fixed styling for textInputs in methods as well as muti-line text updates

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Go to the layout view and multi-select 2 linked blocks, move them together to see the link also moves.

## Agile board tracking

connect to #93 
closes #93
